builder/docker-compose.yml

```diff
@@ -4,6 +4,7 @@ services:
   ubuntu-1604:
     command: ./build.sh
     environment:
+      - BUILD_NO=${BUILD_NUMBER}
```

Comment:
The docker-compose environment is only used for local dev assistance, and the actual build aren't related to a jenkins build in any way (and so don't have a `BUILD_NUMBER` to inherit). Also, it would be best to keep the variable name the same, rather than using `BUILD_NO` since "no" can have a different meaning than "number" and we aren't really trying to save bytes 😉

Reply:
I’ll change the BUILD_NO thing, that’s easy enough to fix.

The packaging script currently uses the build number in place of a software version.

In order to ensure we can still install these packages side-by-side, I’ve moved the official R version number into the packages name and am then using the BUILD_NUMBER as the version number of the package.

So packages comes out with names like:

R-3.6.0-123-amd64.deb

Where ‘r-3.6.0’ is the package name and ‘123’ is the package version. The idea was to allow for updates where the R version hasn’t changed, but maybe we’ve improved the build or packaging processes and re-created the deb/rpm.

If we can’t use the Jenkins BUILD_NUMBER, is there another number we could use in it’s place??

Reply:
Removed the reliance on Jenkins BUILD_NUMBER completely - all built packages are now created with version 1